website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -1,14 +1,10 @@
 # Verifier Contract
 
-To utilize the results generated by a [zkVM program][term-zkvm-program] in your application, it’s essential to [verify][term-verify] the receipt.
-You can verify receipts with a call to a RISC Zero verifier contract.
+To utilize the results generated by a [zkVM program][term-zkvm-program] in your on-chain application, it’s essential to [verify][term-verify] the [receipt][term-receipt] on-chain.
 
-- Every verifier contract implements the [`IRiscZeroVerifier` interface][IRiscZeroVerifier.sol@v0.10], which defines the necessary data structures for the receipt and includes a `verify` method.
-- This interface is implemented by the [`RiscZeroGroth16Verifier` contract][RiscZeroGroth16Verifier.sol@v0.10], a stateless and immutable verifier designed for [RISC Zero Groth16 SNARK proofs][article-groth16].
-
-Using the zkVM, any computation performed off-chain can be proven on-chain using a single verifier contract.
-This simplifies system architecture and eliminates the need for multiple contracts for different circuits.
-Even in the case of a system that needs to use custom circuits, it's possible to prove the resulting custom [receipts][term-receipt] within a [zkVM program][term-zkvm-program] through [proof composition][article-proof-composition] and submit the resulting proof to the verifier contract.
+You can verify a RISC Zero [CompactReceipt][crate-CompactReceipt] using a `verify()` call to the [`RiscZeroVerifierRouter` contract][RiscZeroVerifierRouter@v0.10].
+Your `verify()` call will be routed to the appropriate [base verifier contract](#Addresses), depending on which version of the zkVM was used to generate the receipt.
+By using `RiscZeroVerifierRouter`, you opt in to RISC Zero's [Version Management Design][VersionManagement@main].
```

Comment:
It's possible for apps to deploy their own. I've been referring to the one we deploy as the "managed" option.
```suggestion
By using managed `RiscZeroVerifierRouter`, you opt in to RISC Zero's [Version Management Design][VersionManagement@main].
```